frontend/src/scenes/authentication/signup/SignupContainer.tsx

```diff
@@ -0,0 +1,58 @@
+import { useValues } from 'kea'
+import { preflightLogic } from 'scenes/PreflightCheck/preflightLogic'
+// import { signupLogic } from './signupLogic'
```

Comment:
If we're not using something, I'd just remove it, rather than commenting it out, there's always a risk of forgetting and leaving it there for ages

Reply:
Oops, thanks for finding that